components/raftstore-v2/src/batch/store.rs

```diff
@@ -639,9 +644,22 @@ impl<EK: KvEngine, ER: RaftEngine> StoreSystem<EK, ER> {
                             return;
                         }
                         warn!(logger, "flushing oldest cf of regions {regions:?}");
+                        let flush_count = EK::get_accumulated_flush_count(None).unwrap();
+                        let now = TiInstant::now_coarse();
+                        let total_flush_rate = (flush_count - last_flush.0) as f64
```

Comment:
Please elaborate a bit on the algorithm and what's the benefit of this approach. Thanks.

Reply:
Added a comment.